src/charm_state.py

```diff
@@ -4,7 +4,7 @@
 
 from pydantic import BaseModel, Extra
 
-HTTP_PORT = "443"
+HTTP_PORT = 80
```

Comment:
Is it possible to use a non privileged port?

Reply:
Yes, it is. I've tested with the pollinate client [here](https://pastebin.canonical.com/p/sbPFrXmPhP/)

Reply:
I'll resolve this conversation for now based on that comment above, and we'll address the privileged port issue separately.

Reply:
Actually, per discussions elsewhere it seems there's snap configuration (`http.port`) that can be used to set the port and it's supported by the snap library (see [here](https://github.com/canonical/operator-libs-linux/blob/main/lib/charms/operator_libs_linux/v2/snap.py#L37) for an example).

Reply:
Should that be in the scope of this PR? (ISD-913) I do not have any issues including it here, since it is a simple configuration line

Reply:
I'll tackle it in the reopened PR (https://github.com/canonical/pollen-operator/pull/8), I think it is better that way.